src/util/getTextContentFromHTML.ts

```diff
@@ -1,7 +1,12 @@
+import { isMobile } from './isMobile'
+
 /**
  * Returns text without any html tags for the given HTML value. It accurately represents how user sees the text in the browser.
  */
 export const getTextContentFromHTML = (htmlValue: string) => {
+  // temporary hack as shim createElement is causing to exceeded maximum calls
```

Comment:
can we change the method to do this instead? https://css-tricks.com/snippets/javascript/strip-html-tags-in-javascript/

Reply:
So this would be: 
```js
if (isMobile()) return htmlValue.replace(/(<([^>]+)>)/gi, "")
```